src/index.html

```diff
@@ -1,13 +1,94 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>B&O</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a class="top-bar__logo" href="#">
+          <img src="./images/bo-logo.png" alt="B&O logo">
+        </a>
+
+        <div class="top-bar__icons">
+          <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
```

Comment:
```suggestion
          <a href="tel:+123455555555" class="icon icon--phone"></a>
```
Not fixed from the previous review

Reply:
Thank`s for your check!)